updatecli/updatecli.d/contributor.jenkins.io.expiry.yaml

```diff
@@ -67,5 +67,6 @@ actions:
     scmid: default
     spec:
       title: Bump contributors.jenkins.io azurerm_storage_account_sas `expiry` date {{ source "nextExpiry" }}
+      description: paste this certicate output here (private repo) https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml variable CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING
```

Comment:
```suggestion
      description: paste this certicate output here (private repo) https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml (in the variable `CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING`)
```

Reply:
I would remove the parenthesis from this suggestion.

Reply:
```suggestion
      description: Retrieve the `contributors_jenkins_io_sas_query_string` output to update the `CONTRIBUTORS_JENKINS_IO_FILESHARE_SAS_QUERYSTRING` variable value in https://github.com/jenkins-infra/charts-secrets/blob/main/config/infra.ci.jenkins.io/jenkins-secrets.yaml
```